File: algorithms/1108/1108-defanging-an-ip-address.py
def defangIPaddr(address: str) -> str:
    return address.replace('.', '[.]')


def defangIPaddrOne(address: str) -> str:
    return '[.]'.join(address.split('.'))


if __name__ == "__main__":
    print(defangIPaddr('255.100.50.0'))
    print(defangIPaddrOne('255.100.50.0'))

File: algorithms/1108/1108-defanging-an-ip-address.js
/**
 * @param {string} address
 * @return {string}
 */
var defangIPaddr = function (address) {
    return address.replace(/\./g, "[.]");
};

var defangIPaddrOne = function (address) {
    return (address.split('.')).join('[.]')
};


console.log(defangIPaddr('1.1.1.1'));
console.log(defangIPaddr('255.100.50.0'));
console.log(defangIPaddrOne('1.1.1.1'));
console.log(defangIPaddrOne('255.100.50.0'));